otherlibs/runtime_events/runtime_events.mli

```diff
@@ -147,6 +197,12 @@ module Callbacks : sig
       instrumented runtime. [lost_events] callbacks are called if the consumer
       code detects some unconsumed events have been overwritten.
       *)
+
+  val add : 'a Type.t -> (int -> Timestamp.t -> 'a User.t -> 'a -> unit) -> t
```

Comment:
I'm not fond of naming it just `add`, I feel like it's often better to specify the feature-space being covered in such cases. (like `add_user_event_callback` ?)

But this is a very minor nitpick, feel free to ignore.

Reply:
I see what you mean. I'd settle for `add_user_event`, because the module name is already `Callbacks`, thus providing some context:
```
let callbacks =
    Callbacks.create ()
    |> Callbacks.add_user_event Type.event event_handler
    |> Callbacks.add_user_event Type.counter counter_handler
    |> Callbacks.add_user_event Type.span span_handler
    |> Callbacks.add_user_event custom_type custom_handler
```
or
```
let callbacks =
    let open Callbacks in
    create ()
    |> add_user_event Type.event event_handler
    |> add_user_event Type.counter counter_handler
    |> add_user_event Type.span span_handler
    |> add_user_event custom_type custom_handler
```

Reply:
`add_user_event` sounds pretty nice!